types/d3-scale/index.d.ts

```diff
@@ -87,7 +87,7 @@ export interface ScaleContinuousNumeric<Range, Output> {
      *
      * @param range Array of range values.
      */
-    range(range: Range[]): this;
+    range(range: Range[] | ReadonlyArray<Range>): this;
```

Comment:
Isn't `T[]` assignable to `ReadonlyArray<T>`? I think having just `ReadonlyArray<Range>` here suffices, since the below compiles with "--strict":

```ts
const mutable: string[] = [];
const immutable: ReadonlyArray<string> = mutable;
```

Reply:
I suppose that would be a little more parsimonious. I'll push a commit with the change.